### ISSUE TITLE
Restore ability for users to include custom-lib on the server classpath

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#3590]
In 3.2 custom-lib was isolated into its own resource, and no longer included in the server classpath. However some issues (like MIRTH-3700) still require this as a workaround.

Imported Issue. Original Details:
Jira Issue Key: MIRTH-3703
Reporter: narupley
Created: 2015-04-27T07:04:04.000-0700

**Comment by rbeckman-nextgen** [comment]
Revisions 7669/7670: Added a new setting to mirth.properties to allow libraries in custom-lib to be included with the server classpath on startup. For new installations it is disabled, but it is enabled for migration. Also made changes to ServerMigrator so that comments can be set for individual keys.

Imported Comment. Original Details:
Author: narupley
Created: 2015-04-27T12:24:29.000-0700

**Comment by rbeckman-nextgen** [comment]
Verified that custom jars included within the custom-lib folder are not reachable by code after deselecting all available resources. Verified in newer builds that a property exists to include/exclude the custom-lib folder on the server file path. When the property was false, we deselected all resources and the code was still unreachable by code. After setting the property to true and restarting the server, we were able to call out to the custom jars with no resources.

To test migration, we cleared the classpath property from the mirth.properties in the newest version of MC and connected it to an older version. The property was added to the bottom of the mirth.properties file and set to true.

Imported Comment. Original Details:
Author: jaysenp
Created: 2015-05-22T10:44:00.000-0700
